Makefile

```diff
@@ -1,10 +1,17 @@
-setup:
-	curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
-	pyenv install '2.7.14'
-	pyenv local 2.7.14
-	pyenv virtualenv 2.7.14 samtranslator27
-	pyenv activate samtranslator27
-	pyenv local samtranslator27
+PYENV := $(shell command -v pyenv 2> /dev/null)
+
+install:
+	curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer 2> /dev/null | bash
+
+setup: install
+ifndef PYENV
+    $(error "pyenv was not found, please make sure that pyenv is accessible by adding it to your PATH variable in your bash_profile.")
```

Comment:
Can we generalize this a bit by saying: `make sure pyenv is accessible in your path, (usually by adding to PATH variable in bash_profile, zshrc, or other locations based on your platform)`

I don't want someone to think bash_profile is a requirement when they are using a Z-Shell or Windows

Reply:
Would it be worth add the url for pyenv here as well? So that way, customers can jump right to the pyenv install guide?

Reply:
I will change it and add a link to the docs, good feedback